packages/client/components/BottomNavIconLabel.tsx

```diff
@@ -41,9 +36,31 @@ const Label = styled('div')({
 
 const BottomNavIconLabel = forwardRef((props: Props, ref: Ref<HTMLDivElement>) => {
   const {children, className, icon, iconColor, label} = props
+
+  let Icon = HelpOutline
+  switch (icon) {
+    case 'help_outline':
+      Icon = HelpOutline
+      break
+    case 'timer':
+      Icon = TimerOutlined
+      break
+    case 'event':
+      Icon = Event
+      break
+    case 'person_pin_circle':
+      Icon = PersonPinCircleOutlined
+      break
+  }
+
+  const StyledIcon = styled(Icon)<{iconColor?: string}>(({iconColor}) => ({
+    color: iconColor ? paletteColors[iconColor] : 'inherit',
+    height: 24
+  }))
```

Comment:
-1 Similar as in `BottomControlBarReady`, you should style a div and then you can do
```tsx
      {children ||
        <StyledIcon iconColor={iconColor}>
          <Icon/>
        </StyledIcon>
      }
```